crates/rolldown/tests/esbuild/dce/no_side_effects_comment/artifacts.snap

```diff
@@ -106,7 +106,7 @@ export { f as default };
 ```js
 
 //#region stmt-export-default-before-async-fn-anon.js
-/* #__NO_SIDE_EFFECTS__ */ async function stmt_export_default_before_async_fn_anon_default() {}
+async function stmt_export_default_before_async_fn_anon_default() {}
```

Comment:
@IWANABETHATGUY Can you double check these `__NO_SIDE_EFFECTS__` change?

Reply:
they are expected, https://github.com/rolldown/rolldown/pull/2027 I clarify the reason.